schemas/mobility.schema

```diff
@@ -0,0 +1,25 @@
+SET client_min_messages TO WARNING;
+BEGIN;
+
+DROP SCHEMA IF EXISTS $PG_SCHEMA CASCADE;
+CREATE SCHEMA $PG_SCHEMA_MOBILITY;
+
+CREATE TABLE $PG_SCHEMA_MOBILITY.mobility (
+locale_id  					   INTEGER NOT NULL,
+timestamp  					   TIMESTAMP NOT NULL,
+population_staying_at_home     INTEGER,
+population_not_staying_at_home INTEGER,
+number_of_trips 	    	   INTEGER,
```

Comment:
In the same spirit, any thoughts on going `n_trips` for brevity?

Reply:
I agree these column names are kind of painful. I was considering having multiple rows per locale/date combination and a column called `mobility_feature_id` or something like that then having an additional table in the schema that would provide lookups for the various features.

The downside to that is the table grows longer and becomes less intuitive so I decided against it. If you prefer that approach though I'm open to it, otherwise I'll just shorten up these column names as you suggest.

Reply:
I'd also lean towards going with fewer tables, like you have so far.  I guess in the end it boils down to how often new mobility features are added.  If they're relatively stable, then the downside you've mentioned, and more joins, are probably not worth it.

Reply:
Cool, just updated with shortened column names. I think it's a vast improvement.